.gitignore

```diff
@@ -3,3 +3,4 @@ release/
 .env
 cover.out
 wave
+.idea/
```

Comment:
Never seen this in a `.gitignore` before, what's it for? 🙂

Reply:
it's a directory containing some XML files generated by JetBrains IDEs

Reply:
normally you'd add editor files to your global gitignore rather than every single project you have/use

https://help.github.com/en/github/using-git/ignoring-files#create-a-global-gitignore

Reply:
It's clever. Thanks.
I'll close it.